libs/output-mapping/src/Writer/TableWriter.php

```diff
@@ -283,6 +295,8 @@ private function createDestinationBucket(MappingDestination $destination, array
             TableWriter::SYSTEM_METADATA_PROVIDER,
             $this->getCreatedMetadata($systemMetadata)
         );
+
+        return $this->clientWrapper->getBasicClient()->getBucket($bucketId);
```

Comment:
we don't really need the create method to get the bucket, it can be done upstream

Reply:
not sure what you mean?  like do a `get` after the old `ensure...` method?

Reply:
at least after the create method

Reply:
ok